modules/webbrowser/include/eventhandler.h

```diff
@@ -60,6 +60,9 @@ class EventHandler {
     void setBrowser(const CefRefPtr<CefBrowser>& browser);
     void setBrowserInstance(BrowserInstance* browserInstance);
     void detachBrowser();
+    void touchPressCallback(const double x, const double y);
```

Comment:
No need for the `const` here

Reply:
I agree it's not functionally necessary since these aren't references or pointers, but I like doing this to clearly communicate that the parameters are not to be changed.